disnake/ext/commands/errors.py

```diff
@@ -708,7 +707,7 @@ def __init__(self, missing_roles: SnowflakeList) -> None:
         missing = [f"'{role}'" for role in missing_roles]
 
         if len(missing) > 2:
-            fmt = "{}, or {}".format(", ".join(missing[:-1]), missing[-1])
+            fmt = f"{', '.join(missing[:-1])}, or {missing[-1]}"
```

Comment:
Might be worth making a helper method at the top of the file for these or is it not worth that?

Reply:
Sounds good.
e7d942b803b7dc853936e3cb805a1f801ea779cb

Reply:
Missing L842 and L846